symfony/framework-bundle/3.3/manifest.json

```diff
@@ -15,7 +15,7 @@
         "APP_ENV": "dev",
         "APP_SECRET": "%generate(secret)%",
         "#TRUSTED_PROXIES": "127.0.0.1,127.0.0.2",
-        "#TRUSTED_HOSTS": "'^localhost|example\\.com$'"
+        "#TRUSTED_HOSTS": "'^localhost$|^example\\.com$'"
```

Comment:
better use brackets instead?

Reply:
as explained in the PR description, I'm not sure which is better (w.r.t. the docs and how users will edit it with real values), so if you ask, I answer "please decide" :)

Reply:
I prefer brackets, it's more common

Reply:
:+1: updated (with a simple group, without `?:`, like in the docs and because "more common" too)